engines/sqlite.py

```diff
@@ -99,6 +99,9 @@ def table_exists(self, dbname, tablename):
                 self.existing_table_names.add(line[0].lower())
         return self.table_name(name=tablename, dbname=dbname).lower() in self.existing_table_names
 
+    def to_csv(self):
+        Engine.to_csv(self)
```

Comment:
Seems like this should happen automatically via inheritance. Am I missing something?